applications/adv_networking_bench/cpp/CMakeLists.txt

```diff
@@ -107,3 +101,19 @@ install(
         ../adv_networking_bench_rmax_rx_yaml
   DESTINATION bin/adv_networking_bench/cpp
 )
+
+# Add testing
+if(BUILD_TESTING)
+  file(READ "${CMAKE_CURRENT_SOURCE_DIR}/../adv_networking_bench_default_tx_rx.yaml" TEST_CONFIG)
+  string(REGEX REPLACE "eth_dst_addr:[^\n]*" "eth_dst_addr: notused" TEST_CONFIG ${TEST_CONFIG})
+  string(REGEX REPLACE "ip_src_addr:[^\n]*" "ip_src_addr: notused" TEST_CONFIG ${TEST_CONFIG})
+  string(REGEX REPLACE "ip_dst_addr[^\n]*" "ip_dst_addr: notused" TEST_CONFIG ${TEST_CONFIG})
+  string(REGEX REPLACE "max_duration_ms:[^\n]*" "max_duration_ms: 20000" TEST_CONFIG ${TEST_CONFIG})
+  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/adv_networking_bench_default_tx_rx_test.yaml" ${TEST_CONFIG})
+
+  # Add test
+  file(COPY "testing/adv_networking_test_harness.py" DESTINATION "testing")
+  add_test(NAME adv_networking_bench_dpdk_test
+           COMMAND python3 testing/adv_networking_test_harness.py "./adv_networking_bench adv_networking_bench_default_tx_rx_test.yaml"
```

Comment:
Can you confirm quoted positional parameters are parsed correctly in the script?

```
"./adv_networking_bench adv_networking_bench_default_tx_rx_test.yaml"
```

Reply:
correct, that's the whole command that needs to be passed to `adv_networking_test_harness.py` as a full command string